src/app/core/pages/manage/manage.component.html

```diff
@@ -143,6 +143,11 @@ <h1>Management Dashboard</h1>
 
                             <thead>
                             <tr>
+                                <th class="col-xs-5" (click)="sortUsers('displayTTT')">honorsGeneration<i
```

Comment:
Have you tested what happens if a user's Honors Generation is Null? This is allowed by design.

Reply:
(by the way, this is just something I quickly noticed, I haven't actually started to review yet)